schema.sql

```diff
@@ -1,11 +1,13 @@
 /* Database schema to keep the structure of entire database. */
 
+DROP TABLE IF EXISTS animals
 CREATE TABLE animals
 (
 	id int GENERATED ALWAYS AS IDENTITY,
     name varchar,
     date_of_birth date,
     escape_attempts int,
     neutered boolean,
-    weight_kg decimal
+    weight_kg decimal,
+    species varchar
```

Comment:
- [ ] I think this is not the right way to add a colmun you should use the alter command so you can add a column to an existed table so kindly use the alter command for adding a new column. ✅🙏🏻

You can add somthing like this
> ALTER TABLE animals ADD species varchar(255);

Reply:
I have fixed the issue, thank you so much for the feedback.